### PR TITLE
Unbundle clisops from conda package

### LINKED ISSUE #786: Fix dev installation instructions.

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #786 
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
- [x] `bumpversion (major / minor / patch)` has been called on this branch
- [ ] Tags have been pushed (`git push --tags`)
- [ ] The relevant author information has been added to `.zenodo.json`

### What kind of change does this PR introduce?

* Announces changes to the dependencies of the xclim conda package

### Does this PR introduce a breaking change?

Yes, for conda package users and services using the xclim conda package. PAVICS stack may be affected.

### Other information:

Closing this issue will entail closing https://github.com/Ouranosinc/xclim-feedstock/issues/1

### CODE REVIEW

**Comment by aulemahal** [comment]
Shouldn't we remove clisops from `environment.yml` for this to work, or do I not understand how this works?

File: docs/installation.rst [hunk 1]
```diff
@@ -12,11 +12,13 @@ To install xclim via pip, run this command in your terminal:
 
     $ pip install xclim
 
-To install xclim with spatial subsetting tools (`clisops`_), ensure you have needed system dependencies installed and run:
+To install xclim with spatial subsetting and bias correction tools (`clisops`_), ensure you have the necessary system dependencies installed and run:
```
Comment (aulemahal) [inline docs/installation.rst:15]: bias correction? Hum, I think sdba works without clisops, no?

Comment (Zeitsperre) [reply docs/installation.rst:15]: Right, should that be "regridding"?

Comment (aulemahal) [reply docs/installation.rst:15]: Oh! Yes and no. Indeed regridding tools are coming in clisops. But, I wouldn't recommend them before testing them (I would still point people towards xESMF instead).

**Pull request merged**